logging_metrics_performance/enterprise_logging/enterprise-logging-setup.sh

```diff
@@ -116,6 +114,7 @@ oc delete oauthclient kibana-proxy
 oc new-app logging-deployer-template \
                         --param ES_CLUSTER_SIZE=$CLUSTER_SIZE \
                         --param PUBLIC_MASTER_URL=$PUBLIC_MASTER_URL \
+                        --param KIBANA_HOSTNAME=`hostname` \
```

Comment:
This isn't a good idea, as we might want to point the installer to an existing kibana installation.